### PR TITLE
Fix Powah Crystals being un-meltable

### LINKED ISSUE #3158: Powah Crystals unsmeltable for Tinkers' Construct
### Modpack Version

Enigmatica 6

### Describe your issue

Unable to smelt or craft any tools/weapons from tinkers contructs' extra material list from Powah mod, the crystals

### Crash Report

_No response_

### Latest Log

_No response_

### Have you modified the modpack?

No

### User Modifications

_No response_

### Did the issue happen in singleplayer or on a server?

Both

### PR DESCRIPTION
**Pull request by MuteTiefling**
With the tag being changed from `forge:gems/nitro_crystal` to just `forge:gems/nitro` the rest of the integration needed to be updated as well. I -think- this does it. Seems to be working on my end, anyway.

Should resolve #3158

### CODE REVIEW

File: kubejs/assets/kubejs/lang/en_us.json [hunk 1]
```diff
@@ -186,14 +186,14 @@
     "fluid.kubejs.molten_aeternium": "Molten Aeternium",
     "fluid.kubejs.molten_alfsteel": "Molten Alfsteel",
     "fluid.kubejs.molten_gaia_spirit": "Molten Gaia Spirit",
-    "fluid.kubejs.molten_niotic_crystal": "Molten Niotic Crystal",
-    "fluid.kubejs.molten_nitro_crystal": "Molten Nitro crystal",
-    "fluid.kubejs.molten_spirited_crystal": "Molten Spirited Crystal",
+    "fluid.kubejs.molten_niotic": "Molten Niotic Crystal",
+    "fluid.kubejs.molten_nitro": "Molten Nitro crystal",
```
Comment (DarkAngel22579) [inline kubejs/assets/kubejs/lang/en_us.json:190]: Crystal needs a capital C, somehow missed it in my original pr 😅

Comment (MuteTiefling) [reply kubejs/assets/kubejs/lang/en_us.json:190]: Oh jeez I was trying to find that! Thanks

**Comment by NillerMedDild** [comment]
Thanks! I deleted `kubejs/assets/ponderjs/lang/en_us.json`, hopefully that's okay 😄

**Comment by MuteTiefling** [comment]
> Thanks! I deleted `kubejs/assets/ponderjs/lang/en_us.json`, hopefully that's okay 😄 

Ah yeah sorry. That thing keeps saying it's changed and trying to sneak in.

**Pull request merged**